quirks/password-rules.json

```diff
@@ -89,6 +89,9 @@
     "chase.com": {
         "password-rules": "minlength: 8; maxlength: 32; required: lower, upper; required: digit; required: [!#$%+/=@~]; max-consecutive: 2;"
     },
+    "cigna.com": {
+        "password-rules": "minlength: 8; maxlength: 12; required: digit; required: lower, upper; allowed: [_!.&@]"
```

Comment:
Can you toss a semicolon at the end of this line? Then I think it’ll be good to merge.